src/Lykke.Service.ExchangeConnector/Trading/OrderStatusUpdate.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace TradingBot.Trading
+{
+    public class OrderStatusUpdate
+    {
+        public string ClientOrderId { get; internal set; }
+
+        public string ExchangeOrderId { get; internal set; }
+
+        public bool Success { get; internal set; }
```

Comment:
Is it the same as FailureType or Status?

Reply:
FailureType is filled if Success is false.